testing/cluster_test_utils.py

```diff
@@ -115,6 +115,19 @@ def launch_node(cluster_name, hostname, zk_servers, external_storage, zkid, ip,
 class ClusterTest:
     def setUp(self, num_nodes = 4):
         assert (num_nodes >= 3), ("num_nodes(%s) must be at least 3."%num_nodes)
+
+        # clean out any old docker fixtures
+        docker_containers = docker_client.containers.list(all=True, filters={"name":"ramcloud-node-*"})
+        try:
+            for dc in docker_containers:
+                print("removing container:", dc.name)
+                dc.remove(force=True)
+            docker_network = docker_client.networks.get("ramcloud-net")
+            print("removing network:", docker_network);
+            docker_network.remove()
+        except docker.errors.NotFound as nf:
+            pass
```

Comment:
Please add a comment here explaining that toss a NotFound exception simply means we started with a clean slate, and that's why we pass and press onwards

Reply:
done